src/delta_utils/utils.py

```diff
@@ -0,0 +1,29 @@
+from pathlib import Path
+
+DONT_SEARCH = [
+    "venv",
+    ".git",
+    "__pycache__",
+    "dist",
+    "build",
+    "tests",
+    ".idea",
+    ".mypy_cache",
+    ".pytest_cache",
+]
+
+
+def find_file(filename: str) -> Path:
+    """Find game_mechanics.py."""
```

Comment:
I guess this searches through the current working directory? Could we have a comment about how it finds the file, given that we have dozens of game_mechanics.py?

Reply:
Sure